### ISSUE TITLE
Web-application: In dropdown menu, change the string "CIE xy fundamental chromaticity diagram" into "CIE xy cone-fundamental-based chromaticity diagram"

### ISSUE THREAD

**Issue by janhenrikwold** [ifarup/tc1-82#152]

**Comment by gerardodiego** [comment]
This one is closed. Should I change the text to **CIE xy cone-fundamental-based chromaticity diagram** ?

**Comment by janhenrikwold** [comment]
Yes, and in addition ‘CIE XYZ standard CMFs (Spectrals)’ should be replaced by ‘CIE  XYZ standard colour-matching functions’

One more thing:
The text string ‘Renormalized values’ in front of  checkbox following the drop-down menu should be ‘Renormalised values’, that is british-english language.

- Jan Henrik

Jan Henrik Wold
Høgskolelektor / Associate Professor
Institutt for kommunikasjon og organisasjon / Department of Communication and Organization
Tel.

+47 31008842

Bredalsveien 14


3511 Hønefoss

Jan.Henrik.Wold@hbv.no<mailto:Jan.Henrik.Wold@hbv.no>

Lync

Trykk her<sip:Jan.Henrik.Wold@hbv.no>

www.hbv.no<http://www.hbv.no/>

**Comment by gerardodiego** [comment]
Done and done =)
